docs/scripting/language/Binary.md

```diff
@@ -0,0 +1,385 @@
+---
```

Comment:
This page should be in tutorials and there should be a section at the top explaining it was copied from the forum and the original author was Kyosaur.